templates/KpisList/KpisList.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useMemo } from "react";
+import {
+  HvGrid,
+  HvGlobalActions,
+  HvButton,
+  HvBulkActions,
+  HvPagination,
+} from "@hitachivantara/uikit-react-core";
+import {
+  HvControls,
+  HvLeftControl,
+  HvRightControl,
+  useHvData,
+  useHvSortBy,
+  useHvGlobalFilter,
+  useHvRowSelection,
+  useHvBulkActions,
+  useHvPagination,
+  useHvFilters,
+} from "@hitachivantara/uikit-react-lab";
+import { Refresh } from "@hitachivantara/uikit-react-icons";
+import Kpi from "./Kpi";
+import List from "./List";
+import { getColumns, makeData } from "./makedata";
+
+const actions = [{ id: "refresh", label: "Refresh", icon: <Refresh /> }];
+
+const KpisList = () => {
+  const originalData = useMemo(() => makeData(25), []);
+  const [data] = useState(originalData);
+  const columns = useMemo(() => getColumns(), []);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const instance = useHvData(
+    {
+      data,
+      columns,
+      initialState: { pageSize: 5 },
+    },
+    useHvGlobalFilter,
+    useHvFilters,
+    useHvSortBy,
+    useHvPagination,
+    useHvRowSelection,
+    useHvBulkActions
+  );
+
+  const idsToControl = {
+    cards: "cardsGrid",
+    list: "itemList",
+  };
+
+  // Mock refresh function just simulating some time spent on refetching data
+  const doRefresh = () => {
+    setIsLoading(true);
+    setTimeout(() => setIsLoading(false), 2000);
+    instance.setFilter("status", "");
+  };
+
+  const handleAction = (e, id, action) => {
+    if (action.id === "refresh") {
+      doRefresh();
+    }
+  };
+
+  const bulkActionProps = instance.getHvBulkActionsProps?.();
+
+  const getRequestCount = (status) => {
+    return instance.initialRows.filter((r) => r.original.status === status).length;
+  };
+
+  return (
+    <>
+      <HvGlobalActions title="Requests" backButton={false}>
+        <HvButton category="primary">Request Server</HvButton>
+      </HvGlobalActions>
+
+      <div style={{ marginTop: 40 }}>
+        <HvGrid container>
+          <HvGrid item xl={3} lg={3} md={3} sm={6} xs={12}>
+            <Kpi
+              title="Sucess Requests"
+              count={getRequestCount(0)}
+              color="sema1"
+              variation="up"
+              status={0}
+              instance={instance}
+            />
+          </HvGrid>
+          <HvGrid item xl={3} lg={3} md={3} sm={6} xs={12}>
```

Comment:
the HvGrid breakpoints seems to be the same maybe we could create an object and spread it in the elements